### DISCUSSION TITLE
Admissions?

### DISCUSSION THREAD

**Discussion by aarthur81** [OS4ED/openSIS-Classic#319] [Q&A]
Does the community edition of openSIS not have Admissions as a feature? No admissions form or applicant dashboard? I have 9.1 installed on my own server.

**Comment by openSISAdmin** [comment]
Admissions module is available in openSIS SaaS in the Advanced tier. See the affordable pricing at: https://www.opensis.com/pricing

The Canvas integration is coming later in the year. It will be included in the Advanced Tier.
